### PR TITLE
client cleanup

### PR DESCRIPTION
**Pull request by SuperchupuDev**
- [x] reduce client type/variable duplication
- [x] add loadDirectory/loadFile
- [x] update handlers
- [x] update subcommand handler
- [x] use subcommand builders
- [x] add deploy support

### CODE REVIEW

File: src/classes/Client.ts [hunk 1]
```diff
@@ -23,23 +16,24 @@ import TagHandler from './handlers/TagHandler';
...
+    phishing = new PhishingHandler(this);
+    shop = new ShopHandler(this);
+    tags = new TagHandler(this);
+    cooldown = new Set<string>();
+    ai = new AI(this);
+    invite =
+        'https://discord.com/api/oauth2/authorize?client_id=831932409943425064&scope=bot+applications.commands&permissions=474527689975';
+    version = process.env.npm_package_version;
+    devs = ['707675871355600967', '478823932913516544', '348591272476540928'];
     generating: boolean;
```
Comment (Nyameliaaaa) [inline src/classes/Client.ts]: this is useless i think
image

Comment (SuperchupuDev) [reply src/classes/Client.ts]: @xiboon you are the one who added this what do you think

Comment (hkubus) [reply src/classes/Client.ts]: im pretty sure this was moved to client.ai.generating

Comment (Nyameliaaaa) [reply src/classes/Client.ts]: yep

its isGenerating but ok 👍

**Review by Nyameliaaaa — APPROVED** [review]
👍

**Review by hkubus — APPROVED** [review]
Mom told me to review this

**Pull request merged**